lib/minisign/nacl.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+module Minisign
+  # A module that invokes RbNaCl with user-focused actionable error messages.
+  module NaCl
+    def self.safely
```

Comment:
or instead of a block? something like:

```rb
module Blake2b
  def self.digest(*args)
     assert_libodium_dependency_met!
     RbNaCl::Hash::Blake2b.digest(*args)
  end
end

```